<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <article xmlns="" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
      <head>
         <style type="text/css"> 		 
            	    table {background : #ffffdd;}
            		 	tr {background : #ddddff; padding : 1px;}
            			
         </style>
         <span class="citation_publisher">Public Library of Science</span>
         <h2 class="citation-title">Genome Microevolution of Chikungunya Viruses Causing the Indian Ocean Outbreak</h2>
         <span class="doi">10.1371/journal.pmed.0030263</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Isabelle Schuffenecker</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Isabelle Iteman</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Alain Michault</span>
            <span class="citation_author_institution_ref">3</span>
            <span class="citation_author">Séverine Murri</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Lionel Frangeul</span>
            <span class="citation_author_institution_ref">4</span>
            <span class="citation_author">Marie-Christine Vaney</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Rachel Lavenir</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Nathalie Pardigon</span>
            <span class="citation_author_institution_ref">7</span>
            <span class="citation_author">Jean-Marc Reynes</span>
            <span class="citation_author_institution_ref">8</span>
            <span class="citation_author">François Pettinelli</span>
            <span class="citation_author_institution_ref">9</span>
            <span class="citation_author">Leon Biscornet</span>
            <span class="citation_author_institution_ref">10</span>
            <span class="citation_author">Laure Diancourt</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Stéphanie Michel</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Stéphane Duquerroy</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Ghislaine Guigon</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Marie-Pascale Frenkiel</span>
            <span class="citation_author_institution_ref">7</span>
            <span class="citation_author">Anne-Claire Bréhin</span>
            <span class="citation_author_institution_ref">7</span>
            <span class="citation_author">Nadège Cubito</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Philippe Desprès</span>
            <span class="citation_author_institution_ref">7</span>
            <span class="citation_author">Frank Kunst</span>
            <span class="citation_author_institution_ref">12</span>
            <span class="citation_author">Félix A Rey</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Hervé Zeller</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Sylvain Brisse</span>
            <span class="citation_author_institution_ref">2</span>
            	  CONT
            	  
            <div tagxxx="contrib">
               <name>
                  <span tagx="surname">IV</span>
                  <span tagx="given-names">Herbert Virgin</span>
               </name>
               <div tagxxx="role">Academic Editor</div>
               <span ref-type="aff" rid="edit1"></span>
            </div>
         </div>
         <span class="email">[schuffenecker@cervi-lyon.inserm.fr]</span>
         <span class="conflict">[
            						Competing Interests: The authors have declared that no competing interests exist.
            					]
         </span>
         <span class="contribs">[]</span>
         <span class="citation_pubdate">[2006-5-23]</span>
         <span class="citation_copyright">[Copyright : ]</span>
         <span class="citation_licence">[Licence ]</span>
         <span class="citation_funding">[]</span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div tagxxx="sec">
            <title>Background</title>
            <p>A chikungunya virus outbreak of unprecedented magnitude is currently ongoing in Indian
               Ocean territories. In Réunion Island, this alphavirus has already infected about one-third
               of the human population. The main clinical symptom of the disease is a painful and
               invalidating poly-arthralgia. Besides the arthralgic form, 123 patients with a confirmed
               chikungunya infection have developed severe clinical signs, i.e., neurological signs
               or fulminant hepatitis.
            </p>
         </div>
         <div tagxxx="sec">
            <title>Methods and Findings</title>
            <p>We report the nearly complete genome sequence of six selected viral isolates (isolated
               from five sera and one cerebrospinal fluid), along with partial sequences of glycoprotein
               E1 from a total of 127 patients from Réunion, Seychelles, Mauritius, Madagascar, and
               Mayotte islands. Our results indicate that the outbreak was initiated by a strain
               related to East-African isolates, from which viral variants have evolved following
               a traceable microevolution history. Unique molecular features of the outbreak isolates
               were identified. Notably, in the region coding for the non-structural proteins, ten
               amino acid changes were found, four of which were located in alphavirus-conserved
               positions of nsP2 (which contains helicase, protease, and RNA triphosphatase activities)
               and of the polymerase nsP4. The sole isolate obtained from the cerebrospinal fluid
               showed unique changes in nsP1 (T301I), nsP2 (Y642N), and nsP3 (E460 deletion), not
               obtained from isolates from sera. In the structural proteins region, two noteworthy
               changes (A226V and D284E) were observed in the membrane fusion glycoprotein E1. Homology
               3D modelling allowed mapping of these two changes to regions that are important for
               membrane fusion and virion assembly. Change E1-A226V was absent in the initial strains
               but was observed in &gt;90% of subsequent viral sequences from Réunion, denoting evolutionary
               success possibly due to adaptation to the mosquito vector.
            </p>
         </div>
         <div tagxxx="sec">
            <title>Conclusions</title>
            <p>The unique molecular features of the analyzed Indian Ocean isolates of chikungunya
               virus demonstrate their high evolutionary potential and suggest possible clues for
               understanding the atypical magnitude and virulence of this outbreak.
            </p>
         </div>
      </div>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <p>Viral genome sequence isolated from 6 patients and partial sequences of isolates from
            121 patients at different stages and locations of the outbreak reveals unique and
            evolving genetic features.
         </p>
      </div>
      <div tagxxx="sec">
         <title>Introduction</title>
         <p>Chikungunya virus (CHIKV) is a mosquito-transmitted alphavirus belonging to family
            Togaviridae [
            				
            <span ref-type="bibr" rid="pmed-0030263-b001">
               <sup>
                  <a href="#pmed-0030263-b001">1</a>
               </sup>
            </span>,
            				
            <span ref-type="bibr" rid="pmed-0030263-b002">
               <sup>
                  <a href="#pmed-0030263-b002">2</a>
               </sup>
            </span>]. It was isolated for the first time from a Tanzanian outbreak in 1952 [
            				
            <span ref-type="bibr" rid="pmed-0030263-b003">
               <sup>
                  <a href="#pmed-0030263-b003">3</a>
               </sup>
            </span>]. It is responsible for an acute infection of abrupt onset, characterized by high
            fever, arthralgia, myalgia, headache, and rash [
            				
            <span ref-type="bibr" rid="pmed-0030263-b004">
               <sup>
                  <a href="#pmed-0030263-b004">4</a>
               </sup>
            </span>,
            				
            <span ref-type="bibr" rid="pmed-0030263-b005">
               <sup>
                  <a href="#pmed-0030263-b005">5</a>
               </sup>
            </span>]. Poly-arthralgia, the typical clinical sign of the disease, is very painful. Symptoms
            are generally self-limiting and last 1–10 d. However, arthralgia may persist for months
            or years. In some patients, minor hemorrhagic signs such as epistaxis or gingivorrhagia
            have also been described.
            			
         </p>
         <p>CHIKV is geographically distributed in Africa, India, and South-East Asia. In Africa,
            the virus is maintained through a sylvatic transmission cycle between wild primates
            and mosquitoes such as 
            				
            <i>Aedes luteocephalus, A. furcifer,</i> or 
            				
            <div tagxxx="named-content">A. taylori</div> [
            				
            <span ref-type="bibr" rid="pmed-0030263-b004">
               <sup>
                  <a href="#pmed-0030263-b004">4</a>
               </sup>
            </span>]. In Asia, CHIKV is transmitted from human to human mainly by 
            				
            <div tagxxx="named-content">A. aegypti</div> and, to a lesser extent, by 
            				
            <div tagxxx="named-content">A. albopictus</div> through an urban transmission cycle. Since the 1952 Tanzania outbreak, CHIKV has
            caused outbreaks in East Africa (Tanzania and Uganda), in Austral Africa (Zimbabwe
            and South Africa), in West Africa (Senegal and Nigeria), and in Central Africa (Central
            African Republic and Democratic Republic of the Congo) [
            				
            <span ref-type="bibr" rid="pmed-0030263-b004">
               <sup>
                  <a href="#pmed-0030263-b004">4</a>
               </sup>
            </span>]. The most recent epidemic re-emergence was documented in 1999–2000 in Kinshasa,
            where an estimated 50,000 persons were infected [
            				
            <span ref-type="bibr" rid="pmed-0030263-b006">
               <sup>
                  <a href="#pmed-0030263-b006">6</a>
               </sup>
            </span>]. Since the first documented Asian outbreak in 1958 in Bangkok, Thailand, outbreaks
            have been documented in Thailand, Cambodia, Vietnam, Laos, Myanmar, Malaysia, Philippines,
            and Indonesia [
            				
            <span ref-type="bibr" rid="pmed-0030263-b004">
               <sup>
                  <a href="#pmed-0030263-b004">4</a>
               </sup>
            </span>,
            				
            <span ref-type="bibr" rid="pmed-0030263-b005">
               <sup>
                  <a href="#pmed-0030263-b005">5</a>
               </sup>
            </span>]. The most recent epidemic re-emergence was documented in 2001–2003 in Java, after
            20 y [
            				
            <span ref-type="bibr" rid="pmed-0030263-b007">
               <sup>
                  <a href="#pmed-0030263-b007">7</a>
               </sup>
            </span>]. In both Africa and Asia, the re-emergence was unpredictable, with intervals of
            7–8 y to 20 y between consecutive epidemics.
            			
         </p>
         <p>Since the end of 2004, CHIKV has emerged in the islands of the south-western Indian
            Ocean. Between January and March 2005, more than 5,000 cases were reported in Comoros.
            Later in 2005, the virus has circulated in the other islands, i.e, Mayotte, Seychelles,
            Réunion, and Mauritius. Starting in December 2005, the rainy season gave rise to a
            renewed epidemic circulation of the virus. Since January 1, 2006, several thousands
            cases were reported in each of Mayotte, Mauritius, and Seychelles islands (
            				
            <div tagxxx="ext-link">http://www.invs.sante.fr</div>, 21 April, 2006). The most affected island is Réunion (total population: 770,000),
            with an estimated 244,000 cases (16 April, 2006). More recently, circulation of the
            virus has also been documented in Madagascar and in India.
            			
         </p>
         <p>In Réunion Island, the first documented cases were patients coming back from Comoros
            in March 2005. More than 3,000 cases were reported from March to June [
            				
            <span ref-type="bibr" rid="pmed-0030263-b008">
               <sup>
                  <a href="#pmed-0030263-b008">8</a>
               </sup>
            </span>]. Transmission was limited (50–100 cases per week) during the winter season of the
            southern hemisphere, and a major upsurge was observed from mid-December [
            				
            <span ref-type="bibr" rid="pmed-0030263-b008">
               <sup>
                  <a href="#pmed-0030263-b008">8</a>
               </sup>
            </span>], with an estimated 12,400 cases in 2005 and an estimated 231,600 cases in 2006 (16
            April, 2006). The peak incidence in 2006 was observed during the second week of February,
            with more than 45,000 cases. The number of cases has now decreased, with an estimated
            3,000 cases during the second week of April. Since March 2005, 123 patients with a
            confirmed CHIKV infection have developed severe clinical signs (neurological signs
            or fulminant hepatitis) that justified hospitalization in an intensive care unit.
            Several cases of encephalopathy and major algic syndrome have been associated with
            vertical transmission of the virus (
            				
            <div tagxxx="ext-link">http://www.invs.sante.fr</div>).
            			
         </p>
         <p>CHIKV is an enveloped, positive-strand RNA virus. To date, two CHIKV complete nucleotide
            sequences have been determined, for the strains Ross and S27 [
            				
            <span ref-type="bibr" rid="pmed-0030263-b009">
               <sup>
                  <a href="#pmed-0030263-b009">9</a>
               </sup>
            </span>], both isolated from patients during the 1952 Tanzania outbreak. Another complete
            nucleotide sequence has been determined for a strain isolated in 
            				
            <div tagxxx="named-content">A. furcifer</div> during the Senegal 1983 outbreak (accession no AY726732). Khan et al [
            				
            <span ref-type="bibr" rid="pmed-0030263-b009">
               <sup>
                  <a href="#pmed-0030263-b009">9</a>
               </sup>
            </span>] showed that the S27 genome was similar in its structure to that of other alphaviruses
            and that O'nyong-nyong virus (ONN) was the closest relative to CHIKV. In addition,
            phylogenetic analyses based on partial E1 sequences from African and Asian isolates
            revealed the existence of three distinct CHIKV phylogroups: one containing all isolates
            from West Africa, one containing isolates from Asia, and one corresponding to Eastern,
            Central, and Southern African isolates [
            				
            <span ref-type="bibr" rid="pmed-0030263-b010">
               <sup>
                  <a href="#pmed-0030263-b010">10</a>
               </sup>
            </span>]. Strains isolated in 1999–2000 in the Democratic Republic of Congo belonged to the
            latter phylogroup [
            				
            <span ref-type="bibr" rid="pmed-0030263-b006">
               <sup>
                  <a href="#pmed-0030263-b006">6</a>
               </sup>
            </span>].
            			
         </p>
         <p>In the present study, we determined the nearly complete nucleotide sequences of viruses
            isolated from six patients originating from Réunion and Seychelles islands. In addition,
            partial E1 sequences were determined from sera or cerebrospinal fluid (CSF) from a
            total of 127 patients from Réunion, Seychelles, Madagascar, Mayotte, and Mauritius.
            Our objectives were to determine the genome structure as well as the unique molecular
            features of the Indian Ocean outbreak isolates, which may distinguish them from other
            reported CHIKV and alphavirus sequences. In addition, the phylogenetic origin and
            the diversity and microevolution of the CHIKV strains responsible for the Indian Ocean
            outbreak were investigated.
         </p>
      </div>
      <div tagxxx="sec">
         <title>Methods</title>
         <div tagxxx="sec">
            <h3>Patients</h3>
            <p>The 127 patients for whom partial or complete CHIKV nucleotide sequences were determined
               originated from Réunion (
               					
               <i>n =</i> 89), Seychelles (
               					
               <i>n =</i> 3), Madagascar (
               					
               <i>n =</i> 8), Mayotte (
               					
               <i>n =</i> 23), and Mauritius (
               					
               <i>n =</i> 4). Characteristics of the patients and biological samples are listed in 
               					
               <span ref-type="table" rid="pmed-0030263-t001">
                  <a href="#pmed-0030263-t001">Table 1</a>
               </span>.
               				
            </p>
            <div id="pmed-0030263-t001">
               <label>Table 1</label>
               <caption>
                  <p>Patient Characteristics</p>
               </caption>
               <graphic></graphic>
            </div>
         </div>
         <div tagxxx="sec">
            <h3>Specific Mouse Immune Sera and Indirect Immunofluorescence Assay</h3>
            <p>Mouse hyperimmune ascitic fluid (HMAF) directed against CHIKV was made at the Pasteur
               Institute. HMAFs against French neurotropic virus strain of yellow fever virus, Hawaï
               strain of dengue type-1 virus, or IS-98-ST1 strain of West Nile virus were previously
               described [
               					
               <span ref-type="bibr" rid="pmed-0030263-b011">
                  <sup>
                     <a href="#pmed-0030263-b011">11</a>
                  </sup>
               </span>]. Briefly, specific HMAFs were obtained by repeated immunization of adult mice with
               crude extracts of virus-infected brains from newborn mice followed by the inoculation
               of sarcoma 180. Mouse antisera were collected 1 mo after the first immunization. All
               animal experiments were conducted in accordance with the guidelines of the Office
               Laboratory of Animal Care at the Pasteur Institute.
               				
            </p>
            <p>For indirect immunofluorescence analysis, mosquito cells were fixed with methanol/acetone
               (7 Vol/3 Vol) on glass spots at −20 °C for 20 min. Briefly, the fixed cells were incubated
               with specific HMAFs at a 1:200 dilution in PBS at room temperature for 20 min. After
               extensive washing with PBS, cells were further incubated with FITC-conjugated goat
               anti-mouse IgG antibody (Pierce Biotechnology, Rockford, Illinois, United States)
               at a 1:100 dilution in PBS. The slides were examined using a fluorescence microscope.
            </p>
         </div>
         <div tagxxx="sec">
            <h3>CHIKV Isolation and Viral RNA Extraction</h3>
            <p>CHIKVs were isolated from either human serum or CSF (
               					
               <span ref-type="table" rid="pmed-0030263-t001">
                  <a href="#pmed-0030263-t001">Table 1</a>
               </span>). 
               					
               <div tagxxx="named-content">A. albopictus</div> C6/36 cells were inoculated with 1 ml of serum or CSF diluted 1:10 in Leibovitz-L15
               medium (Invitrogen/Gibco, Carlsbad, California, United States). The cells were grown
               at 28 °C in Leibovitz-L15 medium supplemented with 5% heat-inactivated foetal bovine
               serum (FBS) and 10% tryptose-phosphate. Cells and supernatants were harvested after
               the first passage (5 d) and the second passage (7 d). The virus isolates were identified
               as CHIKV by indirect immunofluorescence using anti-CHIKV HMAF. In the case of clinical
               isolates 05.115, 06.21, 06.27, and 06.49, whose genomes were sequenced, absence of
               yellow fever virus, dengue type-1 virus, and West Nile virus was confirmed by immunofluorescence
               assay using specific HMAF.
               				
            </p>
            <p>Extraction of viral RNA from the CHIKV isolates was performed using the NucleoSpin
               RNA II kit (Machery-Nagel, Düren, Germany) or the QIAAmp Viral Minikit (Qiagen, Courtaboeuf
               Cedex, France) according to manufacturer's recommended procedures. The sequence of
               the non-structural region of isolates 05.115, 06.21, 06.27, and 06.49 was determined
               from RNA extracted from supernatants harvested after the second passage. All other
               CHIKV isolates sequences were obtained using template RNA extracted from the first
               passage. Extraction of viral RNA from biological specimens was performed using the
               QIAAmp Viral Minikit.
            </p>
         </div>
         <div tagxxx="sec">
            <h3>Focus Immunoassay</h3>
            <p>
               <div tagxxx="named-content">A. pseudoscutellaris</div> AP61 cells were grown in 24-well tissue culture plates in Leibovitz L-15 growth medium
               with 10% FBS for 24 h. Mosquito cell monolayers were washed once with Leibovitz L-15,
               and 0.2 ml of Leibovitz L-15/2% FBS was added. Cells were infected with CHIKV in 0.2
               ml of Leibovitz L-15/2% FBS and incubated at 28 °C for 1 h. Overlay medium consisting
               of 0.4 ml of Leibovitz L-15/2% FBS and carboxymethylcellulose (1.6%) was then added,
               and the tissue culture plates were incubated at 28 °C for 2 d. Foci of infected cells
               were visualized by focus immunoassay as previously described [
               					
               <span ref-type="bibr" rid="pmed-0030263-b011">
                  <sup>
                     <a href="#pmed-0030263-b011">11</a>
                  </sup>
               </span>]. The cells were washed with PBS, fixed with 3% paraformaldehyde in PBS for 20 min,
               and permeabilized with 0.5 % Triton X-100 in PBS for 4 min at room temperature. The
               fixed cells were incubated for 20 min at 37 °C with 1:2,000 dilution of HMAF directed
               against CHIKV. Horseradish peroxidase-conjugated goat anti-mouse IgG antibody was
               used as the second antibody (1:100 dilution) at 37 °C for 20 min. Foci were visualized
               with DAB peroxidase substrate (Sigma, St. Louis, Missouri, United States). Computing
               of focus sizes was performed with program AxioVision version 4.5 (Carl Zeiss, Oberkochen,
               Germany) using the automeasure function.
               				
            </p>
         </div>
         <div tagxxx="sec">
            <h3>Nucleotide Sequencing</h3>
            <p>Primers (
               					
               <span ref-type="supplementary-material" rid="st001">Table S1</span>) were designed based on the nucleotide sequence of the S27 strain. RT-PCR was performed
               using the Titan One Tube RT-PCR kit (Roche, Meylan, France). RT-PCR fragments were
               purified by ultrafiltration prior to sequencing (Millipore, Molsheim, France). Sequencing
               reactions were performed using the BigDye Terminator v1.1 cycle sequencing kit (Applied
               Biosystems, Foster City, California, United States) and purified by ethanol precipitation.
               Sequence chromatograms were obtained on automated sequence analysers ABI3100 or ABI3700
               (Applied Biosystems). All amplicons were sequenced on both strands.
               				
            </p>
         </div>
         <div tagxxx="sec">
            <h3>Assembly of Genome Sequences and Sequence Analysis</h3>
            <p>Contig assembly was performed independently by distinct operators and software, using
               either BioNumerics version 4.5 (Applied-Maths, Sint-Martens-Latem, Belgium) or PhredPhrap/Consed
               [
               					
               <span ref-type="bibr" rid="pmed-0030263-b012">
                  <sup>
                     <a href="#pmed-0030263-b012">12</a>
                  </sup>
               </span>]. Both analyses yielded exactly the same consensus sequence for all strains. A single
               contig was obtained for the six isolates. Sequence alignments and computation of substitution
               tables were performed using programs BioNumerics, DNASP version 4.10 [
               					
               <span ref-type="bibr" rid="pmed-0030263-b013">
                  <sup>
                     <a href="#pmed-0030263-b013">13</a>
                  </sup>
               </span>], and DAMBE version 4.2.13 [
               					
               <span ref-type="bibr" rid="pmed-0030263-b014">
                  <sup>
                     <a href="#pmed-0030263-b014">14</a>
                  </sup>
               </span>]. Alignments of nucleotide and amino acid (aa) sequences against selected alphavirus
               sequences were performed with ClustalW version 1.7 [
               					
               <span ref-type="bibr" rid="pmed-0030263-b015">
                  <sup>
                     <a href="#pmed-0030263-b015">15</a>
                  </sup>
               </span>]. Sequence identities were computed with the PHYLIP package [
               					
               <span ref-type="bibr" rid="pmed-0030263-b016">
                  <sup>
                     <a href="#pmed-0030263-b016">16</a>
                  </sup>
               </span>]. RNA secondary structure was predicted with the Vienna RNA secondary structure server
               [
               					
               <span ref-type="bibr" rid="pmed-0030263-b017">
                  <sup>
                     <a href="#pmed-0030263-b017">17</a>
                  </sup>
               </span>]. Neighbour-joining trees were constructed using MEGA version 3.1 [
               					
               <span ref-type="bibr" rid="pmed-0030263-b018">
                  <sup>
                     <a href="#pmed-0030263-b018">18</a>
                  </sup>
               </span>] with the Kimura-2 parameter corrections of multiple substitutions. Reliability of
               nodes was assessed by bootstrap resampling with 1,000 replicates. Amounts of synonymous
               substitutions per synonymous site (Ks) and of non synonymous substitutions per non
               synonymous site (Ka) were estimated using DNASP. RDP2 [
               					
               <span ref-type="bibr" rid="pmed-0030263-b019">
                  <sup>
                     <a href="#pmed-0030263-b019">19</a>
                  </sup>
               </span>] was used to detect putative mosaic sequences.
               				
            </p>
         </div>
         <div tagxxx="sec">
            <h3>3D Structure Modelling</h3>
            <p>The crystallographic structure of the ectodomain of the glycoprotein E1 of Semliki
               Forest virus (SFV) at neutral pH [
               					
               <span ref-type="bibr" rid="pmed-0030263-b020">
                  <sup>
                     <a href="#pmed-0030263-b020">20</a>
                  </sup>
               </span>] was used as a template to model and analyze the two aa mutations of the Indian Ocean
               isolates. The 3D structure figure was prepared using the program RIBBONS [
               					
               <span ref-type="bibr" rid="pmed-0030263-b021">
                  <sup>
                     <a href="#pmed-0030263-b021">21</a>
                  </sup>
               </span>].
               				
            </p>
         </div>
      </div>
      <div tagxxx="sec">
         <title>Results</title>
         <div tagxxx="sec">
            <h3>Genome Structure and Molecular Signatures of the Indian Ocean Outbreak CHIKVs</h3>
            <div tagxxx="sec">
               <h3>Genome organization.</h3>
               <p>We determined the nearly complete genome sequences of six CHIKV isolates (05.115,
                  05.61, 05.209, 06.21, 06.27, and 06.49) representing distinct geographic origins,
                  time points, and clinical forms (
                  						
                  <span ref-type="table" rid="pmed-0030263-t001">
                     <a href="#pmed-0030263-t001">Table 1</a>
                  </span>) of the Indian Ocean outbreak. 11,601 nucleotides were determined, corresponding
                  to positions 52 (5′ non-translated region [NTR]) to 11,667 (3′NTR, end of third repeat
                  sequence element) in the nucleotide sequence of the 1952 Tanzanian isolate S27 (total
                  length 11,826 nt). There were three insertion/deletion events between S27 and Réunion
                  isolates, two of which were observed in the 3′NTR. First, the internal poly-A stretch
                  of 14 nucleotides observed in S27 (11,440–11,443) and corresponding to a probable
                  internal poly-A site [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b009">
                     <sup>
                        <a href="#pmed-0030263-b009">9</a>
                     </sup>
                  </span>] was replaced by a stretch of only 5 A in Indian Ocean isolates, similar to what
                  was observed in other CHIKVs, e.g., the Ross strain. Second, one A was missing in
                  Indian Ocean isolates in a 5-A stretch at S27 position 11,625. Finally, one codon
                  was missing in isolate 06.27, corresponding to nsP3 codon 460, at which all other
                  Indian Ocean isolates analyzed and available alphavirus sequences are GAA, coding
                  for Glu.
                  					
               </p>
               <p>The genome sequences of the six isolates presented in this paper were similar to those
                  previously reported for alphaviruses [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b009">
                     <sup>
                        <a href="#pmed-0030263-b009">9</a>
                     </sup>
                  </span>,
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b022">
                     <sup>
                        <a href="#pmed-0030263-b022">22</a>
                     </sup>
                  </span>,
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b023">
                     <sup>
                        <a href="#pmed-0030263-b023">23</a>
                     </sup>
                  </span>]. Coding sequences consisted of two large open reading frames of 7,422 nt and 3,744
                  nt encoding the non-structural polyprotein (2,474 aa) and the structural polyprotein
                  (1,248 aa), respectively. The non-structural polyprotein is the precursor of proteins
                  nsP1 (535 aa), nsP2 (798 aa), nsP3 (530 aa), and nsP4 (611 aa), and the structural
                  polyprotein is the precursor of proteins C (261 aa), E3 (64 aa), E2 (423 aa), 6K (61
                  aa), and E1 (439 aa). Cleavage sites characteristic of the alphavirus family and glycosylation
                  sites in E3, E2, and E1 were conserved. A 65-nt junction sequence was identified between
                  the stop codon (TAG, 7499–7501) of the non-structural open reading frame and the start
                  codon (7567–7569) of the structural open reading frame. The 5′NTR ended at position
                  76. The 3′NTR region started at position 11,314 and contained three repeat sequence
                  elements with predicted secondary structures (
                  						
                  <span ref-type="supplementary-material" rid="sg001">Figure S1</span>) that were consistent with previous work [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b009">
                     <sup>
                        <a href="#pmed-0030263-b009">9</a>
                     </sup>
                  </span>].
                  					
               </p>
            </div>
            <div tagxxx="sec">
               <h3>Differences between Indian Ocean outbreak isolates and strain S27.</h3>
               <p>Compared to strain S27, Réunion isolate 05.115 (the most closely related to S27; see
                  below) showed 28 aa changes (1.13%) in the non-structural proteins (
                  						
                  <span ref-type="supplementary-material" rid="st002">Table S2</span>), with the highest proportion in nsP3 (2.26%) and the lowest in nsP2 (0.6%). Ten
                  out of 12 aa changes in nsP3 were concentrated between positions 326 and 524 (5.0%
                  variation), similar to findings in ONN viruses [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b024">
                     <sup>
                        <a href="#pmed-0030263-b024">24</a>
                     </sup>
                  </span>]. One important difference with S27 was that the Indian Ocean isolates exhibited
                  an opal stop codon (UGA) at nsP3 codon 524, instead of Arg (CGA) in S27. This opal
                  codon was observed in related alphaviruses [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b009">
                     <sup>
                        <a href="#pmed-0030263-b009">9</a>
                     </sup>
                  </span>,
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b023">
                     <sup>
                        <a href="#pmed-0030263-b023">23</a>
                     </sup>
                  </span>,
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b024">
                     <sup>
                        <a href="#pmed-0030263-b024">24</a>
                     </sup>
                  </span>], and is believed to regulate the expression of nsP4, the putative RNA polymerase,
                  by a read-through mechanism [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b022">
                     <sup>
                        <a href="#pmed-0030263-b022">22</a>
                     </sup>
                  </span>,
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b025">
                     <sup>
                        <a href="#pmed-0030263-b025">25</a>
                     </sup>
                  </span>].
                  					
               </p>
               <p>Compared to S27, the structural proteins of isolate 05.115 showed 21 (1.68%) aa substitutions
                  (
                  						
                  <span ref-type="supplementary-material" rid="st003">Table S3</span>). Notably, envelope protein E2 showed the highest variation, with 14 (3.3%) aa changes,
                  higher than envelope protein E1 (0.68%) and the capsid protein (0.38%). The ratio
                  of rates of evolution of synonymous and non-synonymous sites (Ks/Ka) between S27 and
                  05.115 isolates was 11.0 for the whole polyprotein, whereas it was only 6.12 for protein
                  E2, probably indicative of a positive selection in favour of aa changes in this immunogenic
                  protein. By comparison, Ks/Ka was 18.75 for the non-structural polyprotein.
                  					
               </p>
            </div>
            <div tagxxx="sec">
               <h3>Indian Ocean outbreak molecular signatures in non-structural proteins and phenotypic
                  variation.
               </h3>
               <p>Ten positions (excluding polymorphic positions) had aa that were unique to the non-structural
                  proteins of outbreak isolates, when compared to other CHIKV sequences (
                  						
                  <span ref-type="table" rid="pmed-0030263-t002">
                     <a href="#pmed-0030263-t002">Table 2</a>
                  </span>). Four changes were observed in relatively conserved positions: nsP2–54, nsP2–374,
                  nsP4–254, and nsP4–500. Interestingly, this latter position, which is about 30 aa
                  from the catalytic “GDD” motif, was Leu in the Indian Ocean sequences instead of a
                  Gln in other CHIKV sequences and a strictly conserved Glu in all other alphaviruses.
                  The remaining six changes took place in relatively variable regions (
                  						
                  <span ref-type="table" rid="pmed-0030263-t002">
                     <a href="#pmed-0030263-t002">Table 2</a>
                  </span>).
                  					
               </p>
               <div id="pmed-0030263-t002">
                  <label>Table 2</label>
                  <caption>
                     <p>Relevant Amino Acid Changes Identified between Indian Ocean Isolates versus a Selection
                        of Alphavirus Sequences
                     </p>
                  </caption>
                  <graphic></graphic>
               </div>
               <p>Additional specific changes were observed in isolates 05.209 (nsP3-S358P) and 06.27
                  (nsP1-T301I, nsP2-Y642N, and nsP3-460del). Notably, phenotypic assays showed differences
                  for strain 06.27. Focus immunoassay analysis showed that low-passaged CHIKV 05.115,
                  06.21, 06.27, and 06.49 grown on 
                  						
                  <i>A. albopictus</i> C6/36 (unpublished data) and 
                  						
                  <div tagxxx="named-content">A. pseudoscuterallis</div> AP61 cells (
                  						
                  <span ref-type="supplementary-material" rid="sg002">Figure S2</span>A) formed mixtures of minute, small, and medium-sized foci. Computing analysis revealed
                  that CHIKV 06.27 exhibited larger foci on AP61 cell monolayers as compared with other
                  strains (
                  						
                  <span ref-type="supplementary-material" rid="sg002">Figure S2</span>B). The particular phenotype of CHIKV 06.27 could be attributed to aa differences
                  in the nsPs, which are involved in viral replication [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b022">
                     <sup>
                        <a href="#pmed-0030263-b022">22</a>
                     </sup>
                  </span>]. Studies are under way to determine whether nsP1-T301I, nsP2-Y642N, and nsP3-460del
                  changes may alter the growth of CHIKV 06.27 in mosquito cells.
                  					
               </p>
            </div>
            <div tagxxx="sec">
               <h3>Indian Ocean molecular signatures in structural proteins and 3D modelling.</h3>
               <p>When analyzing the aa sequences of the structural proteins, seven positions (four
                  in E2, one in 6K, and two in E1) were found to be unique to isolates from the Indian
                  Ocean outbreak (
                  						
                  <span ref-type="table" rid="pmed-0030263-t002">
                     <a href="#pmed-0030263-t002">Table 2</a>
                  </span>). Two of these were located in the E2 ectodomain, with Thr 164 and Met 312 being
                  identified in our isolates instead of Ala and Thr, respectively, in all other available
                  CHIKV sequences (
                  						
                  <span ref-type="table" rid="pmed-0030263-t002">
                     <a href="#pmed-0030263-t002">Table 2</a>
                  </span>). The first of these two positions is variable in alphaviruses; it lies in a region
                  previously defined as containing neutralizing epitopes [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b005">
                     <sup>
                        <a href="#pmed-0030263-b005">5</a>
                     </sup>
                  </span>,
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b026">
                     <sup>
                        <a href="#pmed-0030263-b026">26</a>
                     </sup>
                  </span>]. At position 312, Thr is present in other CHIKV, in ONNV, and in SFV, but it varies
                  in other alphaviruses. This position lies in a region identified as important for
                  E1-E2 oligomerization [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b005">
                     <sup>
                        <a href="#pmed-0030263-b005">5</a>
                     </sup>
                  </span>,
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b026">
                     <sup>
                        <a href="#pmed-0030263-b026">26</a>
                     </sup>
                  </span>].
                  					
               </p>
               <p>In E1, two crucial substitutions were observed: one at residue 284, specific to Indian
                  Ocean isolates, and one at residue 226, present in three out of six Indian Ocean isolates
                  (06.21, 06.27, and 06.49). Both mutations were mapped on the 3D structure (modelled
                  from the crystal structure of SFV E1) in 
                  						
                  <span ref-type="fig" rid="pmed-0030263-g001">Figure 1</span>. Interestingly, residue 226 was Ala in all previously reported CHIKV sequences (
                  						
                  <span ref-type="table" rid="pmed-0030263-t002">
                     <a href="#pmed-0030263-t002">Table 2</a>
                  </span>), and was also Ala in the Indian Ocean isolates obtained at the beginning of the
                  outbreak (March and May 2005). Subsequent isolates, obtained from patients in November
                  and December 2005, displayed a Val at this position (see below). Although position
                  226 is relatively variable among alphaviruses, it was observed that a single mutation
                  at this position (Pro to Ser) allowed SFV to adapt to growth in cholesterol-depleted
                  insect cells [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b027">
                     <sup>
                        <a href="#pmed-0030263-b027">27</a>
                     </sup>
                  </span>,
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b028">
                     <sup>
                        <a href="#pmed-0030263-b028">28</a>
                     </sup>
                  </span>].
                  					
               </p>
               <fig>
                  <label>Figure 1</label>
                  <caption>
                     <title>Localization of the E1 Changes on the 3D Structure Modelled from the Crystal Structure
                        of SFV E1
                     </title>
                     <p>(A) Ribbon diagram of E1, with domain I coloured red, domain II yellow, and domain
                        III blue. Green tubes mark the disulfide bonds. The fusion peptide, at the tip of
                        the molecule (in domain II) is coloured orange and labelled. The N-terminus and the
                        C-terminus observed in the crystal (which is 30 aa upstream of the trans-membrane
                        region) are also labelled. The two unique changes observed in the Indian Ocean isolates
                        are indicated by stars and labelled: positions 226 (white) and 284 (magenta).
                     </p>
                     <p>(B) Partial representation (one octant, slightly extended) of the icosahedral E1 scaffold
                        at the surface of the virion, viewed down a 5-fold symmetry axis. One E1 protomer
                        is highlighted in colours, as in (A); all the others are represented in grey. The
                        location of the some of the icosahedral symmetry axes are drawn as solid black symbols:
                        pentagon for 5-fold axis, triangle for 3-fold axes, ellipse for 2-fold axes (which
                        in the T = 4 lattice of alphaviruses are coincident with quasi 6-fold axes). Open
                        triangles indicate roughly the location of the E2 trimers that interact tightly with
                        E1, covering domain II and the fusion peptide, and presenting the main antigenic sites.
                        The open triangles mark also quasi 3-fold symmetry axes of the T = 4 surface icosahedral
                        lattice. A magenta ball marks the location of Glu 284, at an inter-E1 protomer contact
                        site. This contact is propagated 240 times at the surface lattice (note all pink balls
                        drawn on the grey protomers). Note that the fusion peptide, in orange, is pointing
                        up and away from contacts with other E1 protomers. This is more easily seen at the
                        periphery of the virion, where one of them is labelled (FP). In the virion, this region
                        of E1 is not accessible, covered underneath the E2 molecule [
                        								
                        <span ref-type="bibr" rid="pmed-0030263-b020">
                           <sup>
                              <a href="#pmed-0030263-b020">20</a>
                           </sup>
                        </span>].
                        							
                     </p>
                  </caption>
                  <graphic></graphic>
               </fig>
               <p>The other unique aa observed in E1 from Indian Ocean isolates was Glu 284. This is
                  a highly conserved position, which displays an Asp in the majority of alphaviruses
                  or an Asn in SINV (
                  						
                  <span ref-type="table" rid="pmed-0030263-t002">
                     <a href="#pmed-0030263-t002">Table 2</a>
                  </span>). This aa is located at the interface between E1 protomers at the surface of the
                  virion, participating in contacts that make up the icosahedral E1 scaffold (
                  						
                  <span ref-type="fig" rid="pmed-0030263-g001">Figure 1</span>).
                  					
               </p>
            </div>
         </div>
         <div tagxxx="sec">
            <h3>Phylogenetic Analysis</h3>
            <p>Previous work based on E1 protein sequences showed strong phylogeographic structure
               of the CHIKV species [
               					
               <span ref-type="bibr" rid="pmed-0030263-b006">
                  <sup>
                     <a href="#pmed-0030263-b006">6</a>
                  </sup>
               </span>,
               					
               <span ref-type="bibr" rid="pmed-0030263-b010">
                  <sup>
                     <a href="#pmed-0030263-b010">10</a>
                  </sup>
               </span>]. In order to determine the progenitor phylogroup from which the Indian Ocean outbreak
               isolates emerged, we compared a 1,044-nt region within the E1 coding sequence (positions
               271-1314, i.e., codons 91–438) from the six complete genomes with 29 other available
               chikungunya sequences (
               					
               <span ref-type="supplementary-material" rid="st004">Table S4</span>). Phylogenetic analysis (
               					
               <span ref-type="fig" rid="pmed-0030263-g002">Figure 2</span>) clearly demonstrated that the current Indian Ocean isolates represent a homogeneous
               clade within a broad group comprising isolates from East, Central, and South Africa
               (group ECSA, 
               					
               <span ref-type="fig" rid="pmed-0030263-g002">Figure 2</span>). The isolates from the 2000 outbreak in The Democratic of Congo [
               					
               <span ref-type="bibr" rid="pmed-0030263-b006">
                  <sup>
                     <a href="#pmed-0030263-b006">6</a>
                  </sup>
               </span>] also formed a homogeneous clade within group ECSA. There was no ECSA group member
               showing a significantly closer relationship with the Indian Ocean isolates. Asian
               isolates were more distantly related to Indian Ocean isolates and constituted the
               sister group of group ECSA, whereas West-African isolates were even more divergent.
               Inclusion of other alphaviruses, including the closest relative ONN, placed the root
               of the chikungunya isolates on the branch leading to the West-African phylogroup (unpublished
               data).
               				
            </p>
            <fig>
               <label>Figure 2</label>
               <caption>
                  <title>Phylogenetic Relationships among Chikungunya Isolates Based on Partial E1 Nucleotide
                     Sequences
                  </title>
                  <p>Isolates from the Indian Ocean outbreak (Réunion, Seychelles, Mayotte, Mauritius,
                     and Madagascar) represent a distinct clade within a large East-, Central-, and South-African
                     (ECSA) phylogroup. Bootstrap resampling values are indicated at major nodes. The branch
                     leading to West-African phylogroup (of length of approximately 15%) was shortened
                     for convenience. Maximum-likelihood analysis resulted in a very similar phylogeny.
                  </p>
               </caption>
               <graphic></graphic>
            </fig>
            <p>Comparison of the sequences of Indian Ocean outbreak isolates to the S27 sequence
               revealed 316 (2.7%) nucleotide substitutions in isolate 05.115 (
               					
               <span ref-type="supplementary-material" rid="st005">Table S5</span>). The Asian clade Nagpur strain showed 5.1% average nucleotide divergence from 05.115,
               whereas the West-African clade Senegal strain 37997 displayed 15% difference (
               					
               <span ref-type="supplementary-material" rid="st005">Table S5</span>). Interestingly, the latter strain showed complete conservation of an 87-nt portion
               (9,958–10,045, at the junction between structural proteins 6K and E1) with East-African
               and Indian Ocean outbreak isolates. Sequence identity in this portion may reflect
               a past event of genetic recombination between West-African and East-/Central-African
               strains. Differently, we did not find statistical support (
               					
               <i>p</i> &gt; 7 × 10
               					
               <sup>−2</sup>) for sequence mosaicism or recombination since the split between S27 and Réunion
               isolates, although some genomic regions differed in their density of nucleotide polymorphisms.
               				
            </p>
         </div>
         <div tagxxx="sec">
            <h3>Genotypic Variation among Indian Ocean Outbreak Isolates and Microevolutionary Scenario</h3>
            <p>Specific aa changes in the non-structural proteins were observed in the isolates 05.209
               (nsP3-S358P) and 06.27 (nsP1-T301I, nsP2-Y642N, and nsP3-460del). In the structural
               proteins, change E1-A226V was observed in isolates 06.21, 06.27, and 06.49, and change
               E2-Q146R in the Seychelles isolate 05.209. In addition to these non-synonymous changes,
               there were eight silent nucleotide substitutions, observed in 05.209, 06.27, and 06.49
               (
               					
               <span ref-type="table" rid="pmed-0030263-t003">
                  <a href="#pmed-0030263-t003">Table 3</a>
               </span>).
               				
            </p>
            <div id="pmed-0030263-t003">
               <label>Table 3</label>
               <caption>
                  <p>Polymorphisms Observed among Indian Ocean Isolates</p>
               </caption>
               <graphic></graphic>
            </div>
            <p>A history of probable sequence evolution that occurred during the outbreak (
               					
               <span ref-type="fig" rid="pmed-0030263-g003">Figure 3</span>) was deduced from the 14 nucleotide variations observed among the six complete genomes
               (
               					
               <span ref-type="table" rid="pmed-0030263-t003">
                  <a href="#pmed-0030263-t003">Table 3</a>
               </span>). Isolate 05.61 was initially selected for genome analysis because it was isolated
               in March 2005, at the onset of the outbreak, from a Réunion patient returning from
               Comoros Island, where the outbreak had been going on since January 2005. Remarkably,
               the isolates 05.61 and 05.115 (which was the second earliest isolate analyzed), the
               African isolate S27, and previous unrelated chikungunya isolates from Africa and Asia
               were identical at all 14 polymorphic sites. Therefore, the consensus sequence of isolates
               05.61 and 05.115 (consensus sequence 1) likely represents the ancestral genotype of
               the Réunion outbreak. Distribution of the 14 polymorphisms suggested that this founder
               gave rise to three consensus sequences that likely evolved in four steps. First, substitution
               at genome position 10,670 (causing the E1 A226V change) gave rise to consensus sequence
               2, represented by the late-November 2005 isolate 06.21. Second, a G-to-A synonymous
               substitution at position 6,547 (nsP4) led to an intermediate sequence, which itself
               gave rise to two late sequences: consensus sequence 3 (isolate 06.27), following four
               additional substitutions and one codon deletion (
               					
               <span ref-type="table" rid="pmed-0030263-t003">
                  <a href="#pmed-0030263-t003">Table 3</a>
               </span>), and consensus sequence 4 (06.49), which arose after three distinct synonymous substitutions
               (
               					
               <span ref-type="table" rid="pmed-0030263-t003">
                  <a href="#pmed-0030263-t003">Table 3</a>
               </span>). A fifth consensus sequence was represented by the Seychelles isolate 05.209 alone,
               which exhibited four substitutions (two of them causing aa changes in nsP3-S358P and
               in E2-Q146R) compared to consensus sequence 1 (
               					
               <span ref-type="fig" rid="pmed-0030263-g003">Figure 3</span>).
               				
            </p>
            <fig>
               <label>Figure 3</label>
               <caption>
                  <title>Proposed Evolutionary Scenario of CHIKV Isolates from the Indian Ocean Outbreak</title>
                  <p>The scenario is based on six genome sequences determined by direct sequencing of RT-PCR
                     products obtained using RNA extracts as templates; the sequences (Seq.) thus correspond
                     to consensus sequences of the possible mixture of coexisting genomes (quasispecies).
                  </p>
                  <p>Inset: number of cases of E1-226A and E1-226V at different time intervals in Réunion
                     Island, based on partial E1 sequences. E1-226V was observed in consensus sequences
                     2, 3, and 4, and therefore most E1-226V isolates genotyped based on partial E1 sequences
                     are likely related to these genotypes. However, the independent appearance of E1-226V
                     in other genotypes cannot be excluded. The location, size, and relative position of
                     the islands and the African border are indicative. Consensus sequence 1 was obtained
                     from a Réunion patient who travelled back from Comoros in March 2005, and from a Réunion
                     Island patient. Sequences 2–4 were sampled in Réunion Island; sequence 5 was sampled
                     in the Seychelles. The strain codes are given under the corresponding sequences. The
                     question mark in Comoros Archipelago indicates that Sequence 1 possibly originates
                     from Comoros. The dotted link between Sequence 1 and Sequence 5 indicates that the
                     transmission route is hypothetical.
                  </p>
                  <p>CSF, cerebrospinal fluid; Int., deduced intermediate sequence, which was not sampled.</p>
               </caption>
               <graphic></graphic>
            </fig>
            <p>Since Réunion isolates had E1-226A at the beginning of the outbreak and E1-226V later
               in the epidemics, we compared residue 226 in a total of 92 sequences (from 87 sera,
               four CSF, and one amniotic fluid) from 89 patients from Réunion Island. Remarkably,
               the nature of E1–226 differed totally before and after the winter season. Nineteen
               sequences from patients sampled from 16 March to 7 June, 2005 (including the sequence
               originating from a traveller back from Comoros) had E1-226A. Between 7 September and
               end of December 2005, 27 sequences showed E1-226V. Among 46 Réunion sequences from
               2006, E1-226V was observed 40 times (87%) and E1-226A was observed six times (
               					
               <span ref-type="table" rid="pmed-0030263-t001">
                  <a href="#pmed-0030263-t001">Table 1</a>
               </span>). In Madagascar and Seychelles sequences, for which the samples were collected when
               the first clinical cases were suspected (i.e., probably at the beginning of the outbreaks),
               only the E1-226A was observed. On Mayotte, where the outbreak started in early 2005,
               only E1-226V was observed in 23 sequences obtained from patients in 2006. In sequences
               from patients infected in Mauritius, where the outbreak also started in 2005, both
               E1-226A and E1-226V were observed.
               				
            </p>
         </div>
      </div>
      <div tagxxx="sec">
         <title>Discussion</title>
         <p>CHIKV is currently causing one of the largest chikungunya fever outbreaks reported
            in the past 40 y [
            				
            <span ref-type="bibr" rid="pmed-0030263-b004">
               <sup>
                  <a href="#pmed-0030263-b004">4</a>
               </sup>
            </span>–
            				
            <span ref-type="bibr" rid="pmed-0030263-b006">
               <sup>
                  <a href="#pmed-0030263-b006">6</a>
               </sup>
            </span>,
            				
            <span ref-type="bibr" rid="pmed-0030263-b029">
               <sup>
                  <a href="#pmed-0030263-b029">29</a>
               </sup>
            </span>]. The magnitude of the epidemics has surprised the population, policy makers, and
            public health specialists, although chikungunya is not uncommon in nearby regions
            of this part of the world. Our phylogenetic analyses based on partial glycoprotein
            E1 sequences indicate that the Indian Ocean outbreak was caused by the same strain
            on Réunion, Seychelles, Mayotte, Madagascar, and Mauritius islands, and show that
            the outbreak strain is related to East-, Central-, and South-African isolates. Although,
            to our knowledge, no outbreak was reported recently in East Africa, this scenario
            is compatible with the human population exchanges between East Africa and Comoros,
            where the outbreak is believed to have started. Whether the epidemic strain results
            from the evolution of an enzootic strain, as previously described for eastern equine
            encephalitis virus [
            				
            <span ref-type="bibr" rid="pmed-0030263-b030">
               <sup>
                  <a href="#pmed-0030263-b030">30</a>
               </sup>
            </span>], cannot be excluded. Sequencing of additional isolates from recent outbreaks in
            Africa and from possible local reservoirs should define more precisely the origin
            of the Indian Ocean outbreak.
            			
         </p>
         <p>This study represents the first survey, to our knowledge, of intra-outbreak analysis
            of CHIKV nucleotide variation on a genomic scale. The availability of multiple nearly
            complete genome sequences allowed us to deduce the probable history of successive
            molecular evolutionary changes that may have occurred while the outbreak was still
            ongoing. This evolutionary scenario is the most likely based on the six consensus
            sequences obtained, although additional isolates and determination of quasispecies
            heterogeneity are needed to obtain a more precise picture of viral evolution during
            the outbreak.
         </p>
         <p>Whereas E1-226A was the only genotype observed during the first period of the Réunion
            outbreak (March–June 2005), our data show the emergence and predominance of genotype
            E1-226V, which was observed from the beginning of September 2005 and experienced a
            spectacular rise in frequency. The appearance of E1-226V preceded by at least 3 mo
            the explosive epidemic peak of mid-December 2005, and the link between this aa change
            and the rate of transmission thus deserves further investigation. Interestingly, a
            mutation at residue 226 in SFV was observed to release the cholesterol dependence
            of the virus [
            				
            <span ref-type="bibr" rid="pmed-0030263-b027">
               <sup>
                  <a href="#pmed-0030263-b027">27</a>
               </sup>
            </span>,
            				
            <span ref-type="bibr" rid="pmed-0030263-b028">
               <sup>
                  <a href="#pmed-0030263-b028">28</a>
               </sup>
            </span>]. It is thus possible that such a mutation provides a selective advantage to the
            virus in mosquitoes, which are cholesterol auxotrophs. Indeed, in the SFV E1 crystal
            structure, position 226 is located in the ij loop, in contact with the fusion peptide,
            in a region of the protein that is predicted to interact with the target membrane.
            On the other hand, this change may be selectively neutral, and other evolutionary
            factors such as genetic drift or a founder effect could have favoured E1-226V by chance
            alone. In Sindbis virus, a change at position 226 alone was not sufficient to release
            cholesterol dependence [
            				
            <span ref-type="bibr" rid="pmed-0030263-b031">
               <sup>
                  <a href="#pmed-0030263-b031">31</a>
               </sup>
            </span>]. Our data thus point to interesting experiments comparing the cholesterol dependence
            of the isolates differing at position 226 in E1.
            			
         </p>
         <p>To date, only CHIKV laboratory strains, passaged many times on mosquito or mammalian
            cells, had been entirely sequenced [
            				
            <span ref-type="bibr" rid="pmed-0030263-b009">
               <sup>
                  <a href="#pmed-0030263-b009">9</a>
               </sup>
            </span>]. We provide for the first time nearly complete nucleotide sequences of six clinical
            isolates passaged in vitro only once or twice (see 
            				
            <span ref-type="sec" rid="st2">Methods</span>). Limiting the number of passages is crucial because the infecting viral population
            may correspond to a quasispecies [
            				
            <span ref-type="bibr" rid="pmed-0030263-b032">
               <sup>
                  <a href="#pmed-0030263-b032">32</a>
               </sup>
            </span>–
            				
            <span ref-type="bibr" rid="pmed-0030263-b034">
               <sup>
                  <a href="#pmed-0030263-b034">34</a>
               </sup>
            </span>], i.e., a mixed viral population with genotypes co-existing in an equilibrium governed
            by a balance between mutation and natural selection. Repeated in vitro passages may
            act as a filter on this population. For example, the presence in S27 of an Arg codon
            instead of the opal stop codon in Indian Ocean isolates is probably explained by numerous
            in vitro passages of S27, as evolution of opal to Arg was observed experimentally
            in ONN viruses [
            				
            <span ref-type="bibr" rid="pmed-0030263-b024">
               <sup>
                  <a href="#pmed-0030263-b024">24</a>
               </sup>
            </span>]. Whereas it may be advantageous for viral quasispecies to maintain the opal codon
            in vivo, an Arg codon may confer a selective advantage in vitro, as observed for the
            closely related SFV [
            				
            <span ref-type="bibr" rid="pmed-0030263-b035">
               <sup>
                  <a href="#pmed-0030263-b035">35</a>
               </sup>
            </span>]. In the present study, careful inspection of the chromatograms traces identified
            three codons with double peaks observed both on forward and reverse traces (
            				
            <span ref-type="supplementary-material" rid="sg003">Figure S3</span>). In all three cases, alternative bases corresponded to non-synonymous codons that
            may thus alter viral fitness. It is tempting to speculate that quasispecies diversity
            in vivo might facilitate the access to different body sites, such as the central nervous
            system. For example, selection for a subset of genotypes harbouring the changes observed
            in CSF isolate 06.27 may be associated with invasion of the CSF [
            				
            <span ref-type="bibr" rid="pmed-0030263-b034">
               <sup>
                  <a href="#pmed-0030263-b034">34</a>
               </sup>
            </span>]. These observations underscore that the genome sequence of laboratory “reference”
            strains may not accurately reflect the natural situation, as the genotypic complexity
            of quasispecies in vivo is subject to erosion by in vitro selection. Since the Indian
            Ocean isolates sequenced here were subjected to in vitro selection for only a few
            generations, they probably correspond more closely to the in vivo genotypes than previously
            sequenced chikungunya strains.
            			
         </p>
         <p>The aa differences detected among the outbreak isolates may relate to biological or
            pathogenic characteristics of the virus. Although our viral culture results are preliminary,
            they clearly show phenotypic differences between the unique isolate from CSF (06.27),
            isolated from a neonatal encephalopathy case, and three other isolates from sera associated
            with either the classical form of the disease or encephalopathy. The larger foci observed
            in culture with 06.27 could reflect a higher replication rate of the virus and be
            linked to the specific aa changes identified in nsP1, nsP2, and nsP3. Single aa changes
            in nsP1, including a Thr/Ile change (residue 538 of Sindbis virus) [
            				
            <span ref-type="bibr" rid="pmed-0030263-b036">
               <sup>
                  <a href="#pmed-0030263-b036">36</a>
               </sup>
            </span>,
            				
            <span ref-type="bibr" rid="pmed-0030263-b037">
               <sup>
                  <a href="#pmed-0030263-b037">37</a>
               </sup>
            </span>] and a 18-nt deletion in nsP3 have previously been shown to affect neurovirulence
            in other alphaviruses [
            				
            <span ref-type="bibr" rid="pmed-0030263-b036">
               <sup>
                  <a href="#pmed-0030263-b036">36</a>
               </sup>
            </span>–
            				
            <span ref-type="bibr" rid="pmed-0030263-b038">
               <sup>
                  <a href="#pmed-0030263-b038">38</a>
               </sup>
            </span>]. However, in the absence of nsP1, nsP2, and nsP3 structural data, it is difficult
            to predict whether the specific aa changes observed in isolate 06.27 can have a structural
            or functional impact. We also noted that all the viral sequences determined from either
            the serum or the isolates from three neonatal encephalopathy cases and an adult meningo-encephalitis
            case had E1-226V. However, as this genotype is also observed in classical forms of
            the disease, one cannot conclude on a potential link of E1-226V with neuropathogenesis.
            Host factors also have to be considered in the occurrence of neurological forms of
            the disease. For example, the blood–brain crossing may be favoured by age or hypertension.
            			
         </p>
         <p>Molecular signatures of the Indian Ocean outbreak genomes were identified when they
            were compared to all other reported alphavirus sequences. However, these comparisons
            have to be considered with caution because of a potential sampling bias due to the
            small number of previous alphavirus sequences. Nevertheless, these features represent
            interesting targets for future functional studies, as well as for epidemiological
            follow-up. One particularly interesting feature was the E1-226V residue (see above).
            Another interesting molecular signature was E1–284 Asp. Although the pseudo-atomic
            model of the scaffold used is of modest resolution (the resolution of the crystal
            structure is limited—approaching 3Å—and the model results of fitting this structure
            into a 9Å resolution cryo-electron microscopy reconstruction), it appears that the
            side-chain of Asp 284 interacts with the main chain of an adjacent E1 polypeptide
            in the virion. Indeed, it is in a position compatible with acceptance of a hydrogen
            bond from a main chain amide in the contacting E1 protomer. Because the packing is
            very tight (see 
            				
            <span ref-type="fig" rid="pmed-0030263-g001">Figure 1</span>B), it is possible that the longer glutamic acid side chain (which has an extra CH
            				
            <sub>2</sub> group compared to Asp or Asn) may introduce a slight distortion at the contact sites,
            an effect that is propagated by the icosahedral T = 4 symmetry of the virion. Thus,
            a cooperative effect due to this change at position Asp 284 may play a role in either
            allowing a less efficient assembly of new particles in infected cells, or a more efficient
            particle disassembly process during invasion of a new cell, or a combination of both.
            This information can guide new site-directed mutagenesis studies, using reverse genetics,
            to test the effect of the Asp/Glu replacement on the virus cycle.
            			
         </p>
         <p>The magnitude and high political profile of the outbreak has underlined the critical
            lack of knowledge on the biology of CHIKV, contrasting with related model alphaviruses
            such as Sindbis, Semliki Forest, and Ross River. This situation reflects the fact
            that chikungunya infection, despite infecting millions of people since its discovery,
            has been neglected. However, chikungunya disease is clearly responsible for disabling
            and persistent arthralgia, although it remains unresolved whether the symptoms are
            due to persistence of the virus or inappropriate immune response [
            				
            <span ref-type="bibr" rid="pmed-0030263-b039">
               <sup>
                  <a href="#pmed-0030263-b039">39</a>
               </sup>
            </span>,
            				
            <span ref-type="bibr" rid="pmed-0030263-b040">
               <sup>
                  <a href="#pmed-0030263-b040">40</a>
               </sup>
            </span>]. In addition, during the Indian Ocean CHIKV outbreak, a small proportion of the
            patients (about 123 out of 244,000 infected) developed severe clinical signs such
            as neurological signs or hepatitis. Whereas neurovirulence and neuroinvasiveness are
            established for several alphaviruses such as eastern equine encephalitis virus and
            Venezuelan equine encephalitis virus, only two CHIKV strains had previously been isolated
            from children with clinical signs suggestive of encephalitis and meningitis [
            				
            <span ref-type="bibr" rid="pmed-0030263-b041">
               <sup>
                  <a href="#pmed-0030263-b041">41</a>
               </sup>
            </span>,
            				
            <span ref-type="bibr" rid="pmed-0030263-b042">
               <sup>
                  <a href="#pmed-0030263-b042">42</a>
               </sup>
            </span>]. Whether CHIKV Indian Ocean strains have acquired a higher neurovirulence or neuroinvasiveness
            certainly deserves investigation.
            			
         </p>
         <p>In the absence of efficient vaccine or antiviral therapy, vector control is at present
            the only way to limit chikungunya transmission. However, the broad geographic distribution
            of the mosquito vectors 
            				
            <div tagxxx="named-content">A. albopictus</div> and 
            				
            <div tagxxx="named-content">A. aegyptii</div> [
            				
            <span ref-type="bibr" rid="pmed-0030263-b043">
               <sup>
                  <a href="#pmed-0030263-b043">43</a>
               </sup>
            </span>] may allow the expansion of CHIKV to new areas, such as the European or American
            continents. The molecular data reported here on clinical isolates from the current
            epidemics should contribute to bridge the gap of knowledge concerning this human pathogen,
            and will help to provide, in the middle term, more specific and powerful tools to
            combat it.
            			
         </p>
      </div>
      <div tagxxx="sec">
         <title>Supporting Information</title>
         <div tagxxx="supplementary-material">
            <label>Alternative Language Abstract S1</label>
            <caption>
               <title>Translation of the Abstract into French by IS</title>
               <p>(28 KB DOC)</p>
            </caption>
            <div tagxxx="media">
               <caption>
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <label>Figure S1</label>
            <caption>
               <title>Repeat Sequence Elements found in the 3′NTR Region</title>
               <p>(A) Alignment of repeat sequence elements found in the 3′NTR region of CHIKV genome.
                  All sequences form conserved and stable stem-loop structures in which the less conserved
                  nucleotides around position 20 constitute the loop. Three repeat sequence elements
                  are found in all chikungunya genomes. The first one (RSE1) is inserted before the
                  internal poly-A sequence of S27 genome [
                  						
                  <span ref-type="bibr" rid="pmed-0030263-b009">
                     <sup>
                        <a href="#pmed-0030263-b009">9</a>
                     </sup>
                  </span>], whereas the two others are found downstream of this motif. (B) Predicted secondary
                  structure for RSE1 of isolate 05–115.
                  					
               </p>
               <p>(89 KB PPT)</p>
            </caption>
            <div tagxxx="media">
               <caption>
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <label>Figure S2</label>
            <caption>
               <title>Focus Size Phenotype of CHIKV on Mosquito Cells</title>
               <p>Mosquito AP61 cells grown in 24-well plates (10
                  						
                  <sup>5</sup> cells/well) were infected with 5–10 FFU/well or 50–100 FFU/well of low-passaged CHIKV
                  05.115, 06.21, 06.27, and 06.49. Infected cells were overlaid with carboxymethylcellulose
                  in Leibovitz L-15 growth medium with 2% FBS for 2 d to allow focus development at
                  28 °C. Infected cells were fixed with 3% paraformaldehyde in PBS, permeabilized with
                  Triton X-100 in PBS, and foci of CHIK virus replication were immunostained with anti-CHIKV
                  HMAF (dilution 1:2,000) and peroxidase-conjugated goat anti-mouse Ig (dilution 1:100).
                  The histograms depict the total area of viral foci as determined by squared pixels
                  (relative values).
                  					
               </p>
               <p>(2.5 MB PPT)</p>
            </caption>
            <div tagxxx="media">
               <caption>
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <label>Figure S3</label>
            <caption>
               <title>Chromatogram Traces at Three Nucleotide Positions That Showed Mixed Peaks Observed
                  on Both Strands
               </title>
               <p>Such peaks can be explained by the existence of quasispecies, i.e., the coexistence
                  of several distinct genotypes in vivo. Position 978 (nsP1–301) showed a C/T peak (Thr/Ile
                  replacement), in three strains (05.115, 06.21, and 06.49), all three sequenced from
                  passage 2 RNA extracts. Interestingly, this site corresponded to the T substitution
                  in Réunion isolate 06.27 (CSF sample, passage 1) instead of a C in isolate 05.61 and
                  in other CHIKV sequences. Position 1016 showed a mixed A/G peak in strain 06.49, which
                  corresponds to either Met or Val at residue nsP1–313. Finally, position 1070 showed
                  an A/G peak in isolates 05.115 and 06.21, which results in Thr or Ala at residue nsP1–331.
               </p>
               <p>(80 KB PPT)</p>
            </caption>
            <div tagxxx="media">
               <caption>
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <label>Table S1</label>
            <caption>
               <title>Primers Used for RT-PCR and Sequencing</title>
               <p>(20 KB XLS)</p>
            </caption>
            <div tagxxx="media">
               <caption>
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <label>Table S2</label>
            <caption>
               <title>Amino Acid Changes Observed between Strain S27 and Indian Ocean Outbreak Strains in
                  the Non-Structural Proteins
               </title>
               <p>(21 KB XLS)</p>
            </caption>
            <div tagxxx="media">
               <caption>
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <label>Table S3</label>
            <caption>
               <title>Amino Acid Changes Observed in the Structural Genes among Strains S27, Ross, and from
                  the Indian Ocean Outbreak
               </title>
               <p>(19 KB XLS)</p>
            </caption>
            <div tagxxx="media">
               <caption>
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <label>Table S4</label>
            <caption>
               <title>Sequences Used for the Phylogenetic Analysis of Partial E1 Sequences</title>
               <p>(14 KB XLS)</p>
            </caption>
            <div tagxxx="media">
               <caption>
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <label>Table S5</label>
            <caption>
               <title>Sequence Percent Similarity Based on Amino Acids and Nucleotides (in Parentheses)
                  for the Structural (SP) and Non-Structural (NSP) Proteins of Selected Alphaviruses
               </title>
               <p>(17 KB XLS)</p>
            </caption>
            <div tagxxx="media">
               <caption>
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="sec">
            <h3>Accession Numbers</h3>
            <p>The sequences discussed in this paper were deposited in GenBank/EMBL (
               					
               <div tagxxx="ext-link">http://www.ncbi.nlm.nih.gov/Genbank</div>) databases under the accession numbers AM258990–AM258995. The accession numbers for
               CHIKV complete nucleotide sequences discussed in this paper are AF490259 for the Ross
               strain, AF369024 for the S27 strain, and AY726732 for the 
               					
               <div tagxxx="named-content">A. furcifer</div> strain. The Protein Data Bank (
               					
               <div tagxxx="ext-link">http://www.rcsb.org/pdb</div>) accession number for Semliki Forest Virus E1 protein is 2ALA.
               				
            </p>
         </div>
      </div>
      <back>
         <div tagxxx="ack">
            <p>We thank Drs. A. Fourmaintraux, B. Bouhmani, G. Barau, Y. Touret, and P. Tournebize
               from the Groupe Hospitalier Sud-Réunion (St Pierre, Réunion, France) and the sentinel
               physicians of the Observatoire Régional de la Santé (Réunion, France) for providing
               clinical information on patients; and C. Gouyette Plate-forme Synthèse d'Oligonucléotides
               Longs à Haut Débit, Pasteur Genopole Ile-de-France) and C. Bouchier (Plate-forme Génomique,
               Pasteur Genopole Ile-de-France) for help in sequencing.
            </p>
            <p>
               <b>Author contributions.</b> AM, JMR, FP, and LB sent biological samples for virus isolation and sequencing. SM,
               SM and NC isolated virus strains, performed RT-PCRs and viral RNA extractions from
               biological specimens and virus isolates and LF designed and synthesized the primers.
               II, RL, and LD performed RT-PCRs and sequence reactions. II, LF, GG, and SB assembled
               the genome sequence. IS, II, LF, MCV, SD, FAR, and SB analyzed sequence data. MPF,
               ACB, and PD contributed to the production of CHIKV stocks and performed immunofluorescence
               analysis and viral RNA extraction. NP, MPF, and PD performed focus immunoassays. IS,
               PD, FK, FAR, HZ, and SB wrote the manuscript.
               			
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <title>References</title>
               <li tag="ref">
                  <a name="pmed-0030263-b001"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">EG</span>
                        </name>
                        <name>
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">JH</span>
                        </name>
                     </span>
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Schlesinger</span>
                           <span tagx="given-names">S</span>
                        </name>
                        <name>
                           <span tagx="surname">Schlesinger</span>
                           <span tagx="given-names">MJ</span>
                        </name>
                     </span>
                     <span tagx="article-title">Structure and replication of the alphavirus genome</span>
                     <span tagx="source">The Togaviridae and Flaviviridae</span>
                     <span tagx="year">1986</span>
                     <span class="publisher-loc'">[New York]</span>
                     <span class="publisher-name'">[Plenum Press]</span>
                     <span tagx="fpage">35</span>
                     <span tagx="lpage">90</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b002"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Porterfield</span>
                           <span tagx="given-names">JH</span>
                        </name>
                     </span>
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Schlesinger</span>
                           <span tagx="given-names">R</span>
                        </name>
                     </span>
                     <span tagx="article-title">Antigenic characteristics and classification of the 
                        						
                        <i>Togaviridae</i>
                     </span>
                     <span tagx="source">The Togaviruses</span>
                     <span tagx="year">1980</span>
                     <span class="publisher-loc'">[New York]</span>
                     <span class="publisher-name'">[Academic Press]</span>
                     <span tagx="fpage">13</span>
                     <span tagx="lpage">46</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b003"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Ross</span>
                           <span tagx="given-names">RW</span>
                        </name>
                     </span>
                     <span tagx="article-title">The Newala epidemic. III. The virus: Isolation, pathogenic properties and relationship
                        to the epidemic
                     </span>
                     <span tagx="source">J Hyg</span>
                     <span tagx="year">1956</span>
                     <span tagx="volume">54</span>
                     <span tagx="fpage">177</span>
                     <span tagx="lpage">191</span>
                     <span class="pub-id'">[13346078]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b004"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Jupp</span>
                           <span tagx="given-names">PG</span>
                        </name>
                        <name>
                           <span tagx="surname">McIntosh</span>
                           <span tagx="given-names">BM</span>
                        </name>
                     </span>
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Monath</span>
                           <span tagx="given-names">TP</span>
                        </name>
                     </span>
                     <span tagx="article-title">Chikungunya disease</span>
                     <span tagx="source">The Arboviruses: Epidemiology and ecology</span>
                     <span tagx="year">1988</span>
                     <span class="publisher-loc'">[Boca Raton (Florida)]</span>
                     <span class="publisher-name'">[CRC Press]</span>
                     <span tagx="fpage">137</span>
                     <span tagx="lpage">157</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b005"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Johnston</span>
                           <span tagx="given-names">RE</span>
                        </name>
                        <name>
                           <span tagx="surname">Peters</span>
                           <span tagx="given-names">CJ</span>
                        </name>
                     </span>
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Fields</span>
                           <span tagx="given-names">BN</span>
                        </name>
                        <name>
                           <span tagx="surname">Knipe</span>
                           <span tagx="given-names">DM</span>
                        </name>
                        <name>
                           <span tagx="surname">Howley</span>
                           <span tagx="given-names">PM</span>
                        </name>
                     </span>
                     <span tagx="article-title">Alphaviruses associated primarily with fever and polyarthritis</span>
                     <span tagx="source">Fields virology</span>
                     <span tagx="year">1996</span>
                     <span class="publisher-loc'">[Philadelphia]</span>
                     <span class="publisher-name'">[Lippincott-Raven Publishers]</span>
                     <span tagx="fpage">843</span>
                     <span tagx="lpage">898</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b006"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Pastorino</span>
                           <span tagx="given-names">B</span>
                        </name>
                        <name>
                           <span tagx="surname">Muyembe-Tamfum</span>
                           <span tagx="given-names">JJ</span>
                        </name>
                        <name>
                           <span tagx="surname">Bessaud</span>
                           <span tagx="given-names">M</span>
                        </name>
                        <name>
                           <span tagx="surname">Tock</span>
                           <span tagx="given-names">F</span>
                        </name>
                        <name>
                           <span tagx="surname">Tolou</span>
                           <span tagx="given-names">H</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">Epidemic resurgence of Chikungunya virus in democratic Republic of the Congo: Identification
                        of a new central African strain
                     </span>
                     <span tagx="source">J Med Virol</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">74</span>
                     <span tagx="fpage">277</span>
                     <span tagx="lpage">282</span>
                     <span class="pub-id'">[15332277]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b007"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Laras</span>
                           <span tagx="given-names">K</span>
                        </name>
                        <name>
                           <span tagx="surname">Sukri</span>
                           <span tagx="given-names">NC</span>
                        </name>
                        <name>
                           <span tagx="surname">Larasati</span>
                           <span tagx="given-names">RP</span>
                        </name>
                        <name>
                           <span tagx="surname">Bangs</span>
                           <span tagx="given-names">MJ</span>
                        </name>
                        <name>
                           <span tagx="surname">Kosim</span>
                           <span tagx="given-names">R</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">Tracking the re-emergence of epidemic chikungunya virus in Indonesia</span>
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">99</span>
                     <span tagx="fpage">128</span>
                     <span tagx="lpage">141</span>
                     <span class="pub-id'">[15693148]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b008"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Paquet</span>
                           <span tagx="given-names">C</span>
                        </name>
                        <name>
                           <span tagx="surname">Quatresous</span>
                           <span tagx="given-names">I</span>
                        </name>
                        <name>
                           <span tagx="surname">Solet</span>
                           <span tagx="given-names">JL</span>
                        </name>
                        <name>
                           <span tagx="surname">Sissoko</span>
                           <span tagx="given-names">D</span>
                        </name>
                        <name>
                           <span tagx="surname">Renault</span>
                           <span tagx="given-names">P</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">Chikungunya outbreak in Reunion: Epidemiology and surveillance, 2005 to early January
                        2006
                     </span>
                     <span tagx="source">Euro Surveill</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">11</span>
                     <span tagx="fpage">2</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b009"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Khan</span>
                           <span tagx="given-names">AH</span>
                        </name>
                        <name>
                           <span tagx="surname">Morita</span>
                           <span tagx="given-names">K</span>
                        </name>
                        <name>
                           <span tagx="surname">Parquet Md Mdel</span>
                           <span tagx="given-names">C</span>
                        </name>
                        <name>
                           <span tagx="surname">Hasebe</span>
                           <span tagx="given-names">F</span>
                        </name>
                        <name>
                           <span tagx="surname">Mathenge</span>
                           <span tagx="given-names">EG</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">Complete nucleotide sequence of chikungunya virus and evidence for an internal polyadenylation
                        site
                     </span>
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">83</span>
                     <span tagx="fpage">3075</span>
                     <span tagx="lpage">3084</span>
                     <span class="pub-id'">[12466484]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b010"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Powers</span>
                           <span tagx="given-names">AM</span>
                        </name>
                        <name>
                           <span tagx="surname">Brault</span>
                           <span tagx="given-names">AC</span>
                        </name>
                        <name>
                           <span tagx="surname">Tesh</span>
                           <span tagx="given-names">RB</span>
                        </name>
                        <name>
                           <span tagx="surname">Weaver</span>
                           <span tagx="given-names">SC</span>
                        </name>
                     </span>
                     <span tagx="article-title">Re-emergence of Chikungunya and O'nyong-nyong viruses: Evidence for distinct geographical
                        lineages and distant evolutionary relationships
                     </span>
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="year">2000</span>
                     <span tagx="volume">81</span>
                     <span tagx="fpage">471</span>
                     <span tagx="lpage">479</span>
                     <span class="pub-id'">[10644846]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b011"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Navarro-Sanchez</span>
                           <span tagx="given-names">E</span>
                        </name>
                        <name>
                           <span tagx="surname">Altmeyer</span>
                           <span tagx="given-names">MR</span>
                        </name>
                        <name>
                           <span tagx="surname">Amara</span>
                           <span tagx="given-names">A</span>
                        </name>
                        <name>
                           <span tagx="surname">Schwartz</span>
                           <span tagx="given-names">O</span>
                        </name>
                        <name>
                           <span tagx="surname">Fieschi</span>
                           <span tagx="given-names">F</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">DC-SIGN is essential for the productive infection of human dendritic cells by mosquito-cell-derived
                        dengue viruses
                     </span>
                     <span tagx="source">EMBO Rep</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">7</span>
                     <span tagx="fpage">1</span>
                     <span tagx="lpage">6</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b012"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Gordon D Abajian</span>
                           <span tagx="given-names">C</span>
                        </name>
                        <name>
                           <span tagx="surname">Green</span>
                           <span tagx="given-names">P</span>
                        </name>
                     </span>
                     <span tagx="article-title">Consed: A graphical tool for sequence finishing</span>
                     <span tagx="source">Genome Res</span>
                     <span tagx="year">1998</span>
                     <span tagx="volume">8</span>
                     <span tagx="fpage">195</span>
                     <span tagx="lpage">202</span>
                     <span class="pub-id'">[9521923]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b013"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Rozas</span>
                           <span tagx="given-names">J</span>
                        </name>
                        <name>
                           <span tagx="surname">Sanchez-DelBarrio</span>
                           <span tagx="given-names">JC</span>
                        </name>
                        <name>
                           <span tagx="surname">Messeguer</span>
                           <span tagx="given-names">X</span>
                        </name>
                        <name>
                           <span tagx="surname">Rozas</span>
                           <span tagx="given-names">R</span>
                        </name>
                     </span>
                     <span tagx="article-title">DnaSP, DNA polymorphism analyses by the coalescent and other methods</span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">19</span>
                     <span tagx="fpage">2496</span>
                     <span tagx="lpage">2497</span>
                     <span class="pub-id'">[14668244]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b014"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Xia</span>
                           <span tagx="given-names">X</span>
                        </name>
                        <name>
                           <span tagx="surname">Xie</span>
                           <span tagx="given-names">Z</span>
                        </name>
                     </span>
                     <span tagx="article-title">DAMBE: Software package for data analysis in molecular biology and evolution</span>
                     <span tagx="source">J Hered</span>
                     <span tagx="year">2001</span>
                     <span tagx="volume">92</span>
                     <span tagx="fpage">371</span>
                     <span tagx="lpage">373</span>
                     <span class="pub-id'">[11535656]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b015"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Thompson</span>
                           <span tagx="given-names">JD</span>
                        </name>
                        <name>
                           <span tagx="surname">Higgins</span>
                           <span tagx="given-names">DG</span>
                        </name>
                        <name>
                           <span tagx="surname">Gibson</span>
                           <span tagx="given-names">TJ</span>
                        </name>
                     </span>
                     <span tagx="article-title">CLUSTAL W: Improving the sensitivity of progressive multiple sequence alignment through
                        sequence weighting, positions-specific gap penalties and weight matrix choice
                     </span>
                     <span tagx="source">Nucleic Acids Res</span>
                     <span tagx="year">1994</span>
                     <span tagx="volume">22</span>
                     <span tagx="fpage">4673</span>
                     <span tagx="lpage">4680</span>
                     <span class="pub-id'">[7984417]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b016"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Felsenstein</span>
                           <span tagx="given-names">J</span>
                        </name>
                     </span>
                     <span tagx="article-title">PHYLIP—Phylogeny Inference Package (version 3.2)</span>
                     <span tagx="source">Cladistics</span>
                     <span tagx="year">1989</span>
                     <span tagx="volume">5</span>
                     <span tagx="fpage">164</span>
                     <span tagx="lpage">166</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b017"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Hofacker</span>
                           <span tagx="given-names">IL</span>
                        </name>
                     </span>
                     <span tagx="article-title">Vienna RNA secondary structure server</span>
                     <span tagx="source">Nucleic Acids Res</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">31</span>
                     <span tagx="fpage">3429</span>
                     <span tagx="lpage">3431</span>
                     <span class="pub-id'">[12824340]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b018"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Kumar</span>
                           <span tagx="given-names">S</span>
                        </name>
                        <name>
                           <span tagx="surname">Tamura</span>
                           <span tagx="given-names">K</span>
                        </name>
                        <name>
                           <span tagx="surname">Nei</span>
                           <span tagx="given-names">M</span>
                        </name>
                     </span>
                     <span tagx="article-title">MEGA3: Integrated software for molecular evolutionary genetics analysis and sequence
                        alignment
                     </span>
                     <span tagx="source">Brief Bioinform</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">5</span>
                     <span tagx="fpage">150</span>
                     <span tagx="lpage">163</span>
                     <span class="pub-id'">[15260895]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b019"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Martin</span>
                           <span tagx="given-names">DP</span>
                        </name>
                        <name>
                           <span tagx="surname">Williamson</span>
                           <span tagx="given-names">C</span>
                        </name>
                        <name>
                           <span tagx="surname">Posada</span>
                           <span tagx="given-names">D</span>
                        </name>
                     </span>
                     <span tagx="article-title">RDP2: Recombination detection and analysis from sequence alignments</span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">21</span>
                     <span tagx="fpage">260</span>
                     <span tagx="lpage">262</span>
                     <span class="pub-id'">[15377507]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b020"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Roussel</span>
                           <span tagx="given-names">A</span>
                        </name>
                        <name>
                           <span tagx="surname">Lescar</span>
                           <span tagx="given-names">J</span>
                        </name>
                        <name>
                           <span tagx="surname">Vaney</span>
                           <span tagx="given-names">MC</span>
                        </name>
                        <name>
                           <span tagx="surname">Wengler</span>
                           <span tagx="given-names">G</span>
                        </name>
                        <name>
                           <span tagx="surname">Wengler</span>
                           <span tagx="given-names">G</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">Structure and interactions at the viral surface of the envelope protein E1 of semliki
                        forest virus
                     </span>
                     <span tagx="source">Structure</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">14</span>
                     <span tagx="fpage">75</span>
                     <span tagx="lpage">86</span>
                     <span class="pub-id'">[16407067]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b021"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Carson</span>
                           <span tagx="given-names">M</span>
                        </name>
                     </span>
                     <span tagx="article-title">Ribbon models of macromolecules</span>
                     <span tagx="source">J Mol Graph</span>
                     <span tagx="year">1987</span>
                     <span tagx="volume">5</span>
                     <span tagx="fpage">103</span>
                     <span tagx="lpage">106</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b022"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">JH</span>
                        </name>
                        <name>
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">EG</span>
                        </name>
                     </span>
                     <span tagx="article-title">The alphaviruses: Gene expression, replication, and evolution</span>
                     <span tagx="source">Microbiol Rev</span>
                     <span tagx="year">1994</span>
                     <span tagx="volume">58</span>
                     <span tagx="fpage">491</span>
                     <span tagx="lpage">562</span>
                     <span class="pub-id'">[7968923]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b023"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Lavergne</span>
                           <span tagx="given-names">A</span>
                        </name>
                        <name>
                           <span tagx="surname">Thoisy</span>
                           <span tagx="given-names">BD</span>
                        </name>
                        <name>
                           <span tagx="surname">Lacoste</span>
                           <span tagx="given-names">V</span>
                        </name>
                        <name>
                           <span tagx="surname">Pascalis</span>
                           <span tagx="given-names">H</span>
                        </name>
                        <name>
                           <span tagx="surname">Pouliquen</span>
                           <span tagx="given-names">JF</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">Mayaro virus: Complete nucleotide sequence and phylogenetic relationships with other
                        alphaviruses
                     </span>
                     <span tagx="source">Virus Res</span>
                     <span tagx="year">2005</span>
                     <div tagxxx="comment">In press</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b024"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Lanciotti</span>
                           <span tagx="given-names">RS</span>
                        </name>
                        <name>
                           <span tagx="surname">Ludwig</span>
                           <span tagx="given-names">ML</span>
                        </name>
                        <name>
                           <span tagx="surname">Rwaguma</span>
                           <span tagx="given-names">EB</span>
                        </name>
                        <name>
                           <span tagx="surname">Lutwama</span>
                           <span tagx="given-names">JJ</span>
                        </name>
                        <name>
                           <span tagx="surname">Kram</span>
                           <span tagx="given-names">TM</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">Emergence of epidemic O'nyong-nyong fever in Uganda after a 35-year absence: Genetic
                        characterization of the virus
                     </span>
                     <span tagx="source">Virology</span>
                     <span tagx="year">1998</span>
                     <span tagx="volume">252</span>
                     <span tagx="fpage">258</span>
                     <span tagx="lpage">268</span>
                     <span class="pub-id'">[9875334]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b025"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">EG</span>
                        </name>
                        <name>
                           <span tagx="surname">Levinson</span>
                           <span tagx="given-names">R</span>
                        </name>
                        <name>
                           <span tagx="surname">Rice</span>
                           <span tagx="given-names">CM</span>
                        </name>
                        <name>
                           <span tagx="surname">Dalrymple</span>
                           <span tagx="given-names">J</span>
                        </name>
                        <name>
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">JH</span>
                        </name>
                     </span>
                     <span tagx="article-title">Nonstructural proteins nsP3 and nsP4 of Ross River and O'Nyong-nyong viruses: Sequence
                        and comparison with those of other alphaviruses
                     </span>
                     <span tagx="source">Virology</span>
                     <span tagx="year">1988</span>
                     <span tagx="volume">164</span>
                     <span tagx="fpage">265</span>
                     <span tagx="lpage">274</span>
                     <span class="pub-id'">[2834873]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b026"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Griffin</span>
                           <span tagx="given-names">DE</span>
                        </name>
                     </span>
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Knipe</span>
                           <span tagx="given-names">DM</span>
                        </name>
                        <name>
                           <span tagx="surname">Howley</span>
                           <span tagx="given-names">PM</span>
                        </name>
                     </span>
                     <span tagx="article-title">Alphaviruses</span>
                     <span tagx="source">Fields virology</span>
                     <span tagx="year">2001</span>
                     <span class="publisher-loc'">[Philadelphia]</span>
                     <span class="publisher-name'">[Lippincott Williams &amp; Wilkins]</span>
                     <span tagx="fpage">917</span>
                     <span tagx="lpage">962</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b027"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Vashishtha</span>
                           <span tagx="given-names">M</span>
                        </name>
                        <name>
                           <span tagx="surname">Phalen</span>
                           <span tagx="given-names">T</span>
                        </name>
                        <name>
                           <span tagx="surname">Marquardt</span>
                           <span tagx="given-names">MT</span>
                        </name>
                        <name>
                           <span tagx="surname">Ryu</span>
                           <span tagx="given-names">JS</span>
                        </name>
                        <name>
                           <span tagx="surname">Ng</span>
                           <span tagx="given-names">AC</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">A single point mutation controls the cholesterol dependence of Semliki Forest virus
                        entry and exit
                     </span>
                     <span tagx="source">J Cell Biol</span>
                     <span tagx="year">1998</span>
                     <span tagx="volume">140</span>
                     <span tagx="fpage">91</span>
                     <span tagx="lpage">99</span>
                     <span class="pub-id'">[9425157]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b028"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Ahn</span>
                           <span tagx="given-names">A</span>
                        </name>
                        <name>
                           <span tagx="surname">Schoepp</span>
                           <span tagx="given-names">RJ</span>
                        </name>
                        <name>
                           <span tagx="surname">Sternberg</span>
                           <span tagx="given-names">D</span>
                        </name>
                        <name>
                           <span tagx="surname">Kielian</span>
                           <span tagx="given-names">M</span>
                        </name>
                     </span>
                     <span tagx="article-title">Growth and stability of a cholesterol-independent Semliki Forest virus mutant in mosquitoes</span>
                     <span tagx="source">Virology</span>
                     <span tagx="year">1999</span>
                     <span tagx="volume">262</span>
                     <span tagx="fpage">452</span>
                     <span tagx="lpage">456</span>
                     <span class="pub-id'">[10502523]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b029"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Williams</span>
                           <span tagx="given-names">MC</span>
                        </name>
                        <name>
                           <span tagx="surname">Woodall</span>
                           <span tagx="given-names">JP</span>
                        </name>
                        <name>
                           <span tagx="surname">Corbet</span>
                           <span tagx="given-names">PS</span>
                        </name>
                        <name>
                           <span tagx="surname">Gillett</span>
                           <span tagx="given-names">JD</span>
                        </name>
                     </span>
                     <span tagx="article-title">O'nyong-Nyong Fever: An epidemic virus disease in East Africa. 8. Virus isolations
                        from Anopheles mosquitoes
                     </span>
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="year">1965</span>
                     <span tagx="volume">59</span>
                     <span tagx="fpage">300</span>
                     <span tagx="lpage">306</span>
                     <span class="pub-id'">[14298035]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b030"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Weaver</span>
                           <span tagx="given-names">SC</span>
                        </name>
                        <name>
                           <span tagx="surname">Barrett</span>
                           <span tagx="given-names">AD</span>
                        </name>
                     </span>
                     <span tagx="article-title">Transmission cycles, host range, evolution and emergence of arboviral disease</span>
                     <span tagx="source">Nat Rev Microbiol</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">2</span>
                     <span tagx="fpage">789</span>
                     <span tagx="lpage">801</span>
                     <span class="pub-id'">[15378043]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b031"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Lu</span>
                           <span tagx="given-names">YE</span>
                        </name>
                        <name>
                           <span tagx="surname">Cassese</span>
                           <span tagx="given-names">T</span>
                        </name>
                        <name>
                           <span tagx="surname">Kielian</span>
                           <span tagx="given-names">M</span>
                        </name>
                     </span>
                     <span tagx="article-title">The cholesterol requirement for sindbis virus entry and exit and characterization
                        of a spike protein region involved in cholesterol dependence
                     </span>
                     <span tagx="source">J Virol</span>
                     <span tagx="year">1999</span>
                     <span tagx="volume">73</span>
                     <span tagx="fpage">4272</span>
                     <span tagx="lpage">4278</span>
                     <span class="pub-id'">[10196324]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b032"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Holland</span>
                           <span tagx="given-names">J</span>
                        </name>
                        <name>
                           <span tagx="surname">Spindler</span>
                           <span tagx="given-names">K</span>
                        </name>
                        <name>
                           <span tagx="surname">Horodyski</span>
                           <span tagx="given-names">F</span>
                        </name>
                        <name>
                           <span tagx="surname">Grabau</span>
                           <span tagx="given-names">E</span>
                        </name>
                        <name>
                           <span tagx="surname">Nichol</span>
                           <span tagx="given-names">S</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">Rapid evolution of RNA genomes</span>
                     <span tagx="source">Science</span>
                     <span tagx="year">1982</span>
                     <span tagx="volume">215</span>
                     <span tagx="fpage">1577</span>
                     <span tagx="lpage">1585</span>
                     <span class="pub-id'">[7041255]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b033"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Domingo</span>
                           <span tagx="given-names">E</span>
                        </name>
                        <name>
                           <span tagx="surname">Holland</span>
                           <span tagx="given-names">JJ</span>
                        </name>
                     </span>
                     <span tagx="article-title">RNA virus mutations and fitness for survival</span>
                     <span tagx="source">Annu Rev Microbiol</span>
                     <span tagx="year">1997</span>
                     <span tagx="volume">51</span>
                     <span tagx="fpage">151</span>
                     <span tagx="lpage">178</span>
                     <span class="pub-id'">[9343347]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b034"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Vignuzzi</span>
                           <span tagx="given-names">M</span>
                        </name>
                        <name>
                           <span tagx="surname">Stone</span>
                           <span tagx="given-names">JK</span>
                        </name>
                        <name>
                           <span tagx="surname">Arnold</span>
                           <span tagx="given-names">JJ</span>
                        </name>
                        <name>
                           <span tagx="surname">Cameron</span>
                           <span tagx="given-names">CE</span>
                        </name>
                        <name>
                           <span tagx="surname">Andino</span>
                           <span tagx="given-names">R</span>
                        </name>
                     </span>
                     <span tagx="article-title">Quasispecies diversity determines pathogenesis through cooperative interactions in
                        a viral population
                     </span>
                     <span tagx="source">Nature</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">439</span>
                     <span tagx="fpage">344</span>
                     <span tagx="lpage">348</span>
                     <span class="pub-id'">[16327776]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b035"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Kim</span>
                           <span tagx="given-names">KH</span>
                        </name>
                        <name>
                           <span tagx="surname">Rumenapf</span>
                           <span tagx="given-names">T</span>
                        </name>
                        <name>
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">EG</span>
                        </name>
                        <name>
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">JH</span>
                        </name>
                     </span>
                     <span tagx="article-title">Regulation of Semliki Forest virus RNA replication: A model for the control of alphavirus
                        pathogenesis in invertebrate hosts
                     </span>
                     <span tagx="source">Virology</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">323</span>
                     <span tagx="fpage">153</span>
                     <span tagx="lpage">163</span>
                     <span class="pub-id'">[15165827]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b036"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Heise</span>
                           <span tagx="given-names">C</span>
                        </name>
                        <name>
                           <span tagx="surname">Kirn</span>
                           <span tagx="given-names">DH</span>
                        </name>
                     </span>
                     <span tagx="article-title">Replication-selective adenoviruses as oncolytic agents</span>
                     <span tagx="source">J Clin Invest</span>
                     <span tagx="year">2000</span>
                     <span tagx="volume">105</span>
                     <span tagx="fpage">847</span>
                     <span tagx="lpage">851</span>
                     <span class="pub-id'">[10749561]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b037"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Heise</span>
                           <span tagx="given-names">MT</span>
                        </name>
                        <name>
                           <span tagx="surname">White</span>
                           <span tagx="given-names">LJ</span>
                        </name>
                        <name>
                           <span tagx="surname">Simpson</span>
                           <span tagx="given-names">DA</span>
                        </name>
                        <name>
                           <span tagx="surname">Leonard</span>
                           <span tagx="given-names">C</span>
                        </name>
                        <name>
                           <span tagx="surname">Bernard</span>
                           <span tagx="given-names">KA</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">An attenuating mutation in nsP1 of the Sindbis-group virus S.A.AR86 accelerates nonstructural
                        protein processing and up-regulates viral 26S RNA synthesis
                     </span>
                     <span tagx="source">J Virol</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">77</span>
                     <span tagx="fpage">1149</span>
                     <span tagx="lpage">1156</span>
                     <span class="pub-id'">[12502831]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b038"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Suthar</span>
                           <span tagx="given-names">MS</span>
                        </name>
                        <name>
                           <span tagx="surname">Shabman</span>
                           <span tagx="given-names">R</span>
                        </name>
                        <name>
                           <span tagx="surname">Madric</span>
                           <span tagx="given-names">K</span>
                        </name>
                        <name>
                           <span tagx="surname">Lambeth</span>
                           <span tagx="given-names">C</span>
                        </name>
                        <name>
                           <span tagx="surname">Heise</span>
                           <span tagx="given-names">MT</span>
                        </name>
                     </span>
                     <span tagx="article-title">Identification of adult mouse neurovirulence determinants of the Sindbis virus strain
                        AR86
                     </span>
                     <span tagx="source">J Virol</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">79</span>
                     <span tagx="fpage">4219</span>
                     <span tagx="lpage">4228</span>
                     <span class="pub-id'">[15767423]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b039"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Condon</span>
                           <span tagx="given-names">RJ</span>
                        </name>
                        <name>
                           <span tagx="surname">Rouse</span>
                           <span tagx="given-names">IL</span>
                        </name>
                     </span>
                     <span tagx="article-title">Acute symptoms and sequelae of Ross River virus infection in South-Western Australia:
                        A follow-up study
                     </span>
                     <span tagx="source">Clin Diagn Virol</span>
                     <span tagx="year">1995</span>
                     <span tagx="volume">3</span>
                     <span tagx="fpage">273</span>
                     <span tagx="lpage">284</span>
                     <span class="pub-id'">[15566808]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b040"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Selden</span>
                           <span tagx="given-names">SM</span>
                        </name>
                        <name>
                           <span tagx="surname">Cameron</span>
                           <span tagx="given-names">AS</span>
                        </name>
                     </span>
                     <span tagx="article-title">Changing epidemiology of Ross River virus disease in South Australia</span>
                     <span tagx="source">Med J Aust</span>
                     <span tagx="year">1996</span>
                     <span tagx="volume">165</span>
                     <span tagx="fpage">313</span>
                     <span tagx="lpage">317</span>
                     <span class="pub-id'">[8862330]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b041"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Mazaud</span>
                           <span tagx="given-names">R</span>
                        </name>
                        <name>
                           <span tagx="surname">Salaün</span>
                           <span tagx="given-names">JJ</span>
                        </name>
                        <name>
                           <span tagx="surname">Montabone</span>
                           <span tagx="given-names">H</span>
                        </name>
                        <name>
                           <span tagx="surname">Goube</span>
                           <span tagx="given-names">P</span>
                        </name>
                        <name>
                           <span tagx="surname">Bazillio</span>
                           <span tagx="given-names">R</span>
                        </name>
                     </span>
                     <span tagx="article-title">Troubles neurologiques et sensoriels aigus dans la dengue et la fièvre à Chikungunya</span>
                     <span tagx="source">Bull Soc Pathol Exot</span>
                     <span tagx="year">1971</span>
                     <span tagx="volume">64</span>
                     <span tagx="fpage">22</span>
                     <span tagx="lpage">30</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b042"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Nimmannitya</span>
                           <span tagx="given-names">S</span>
                        </name>
                        <name>
                           <span tagx="surname">Halstead</span>
                           <span tagx="given-names">SB</span>
                        </name>
                        <name>
                           <span tagx="surname">Cohen</span>
                           <span tagx="given-names">SN</span>
                        </name>
                        <name>
                           <span tagx="surname">Margiotta</span>
                           <span tagx="given-names">MR</span>
                        </name>
                     </span>
                     <span tagx="article-title">Dengue and chikungunya virus infection in man in Thailand, 1962–1964. I. Observations
                        on hospitalized patients with hemorrhagic fever
                     </span>
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="year">1969</span>
                     <span tagx="volume">18</span>
                     <span tagx="fpage">954</span>
                     <span tagx="lpage">971</span>
                     <span class="pub-id'">[5355242]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b043"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Gratz</span>
                           <span tagx="given-names">NG</span>
                        </name>
                     </span>
                     <span tagx="article-title">Critical review of the vector status of 
                        						
                        <div tagxxx="named-content">Aedes albopictus</div>
                     </span>
                     <span tagx="source">Med Vet Entomol</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">18</span>
                     <span tagx="fpage">215</span>
                     <span tagx="lpage">227</span>
                     <span class="pub-id'">[15347388]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pmed-0030263-b044"></a>
                  <div tagxxx="citation">
                     <span class="person-group'">
                        <name>
                           <span tagx="surname">Lescar</span>
                           <span tagx="given-names">J</span>
                        </name>
                        <name>
                           <span tagx="surname">Roussel</span>
                           <span tagx="given-names">A</span>
                        </name>
                        <name>
                           <span tagx="surname">Wien</span>
                           <span tagx="given-names">MW</span>
                        </name>
                        <name>
                           <span tagx="surname">Navaza</span>
                           <span tagx="given-names">J</span>
                        </name>
                        <name>
                           <span tagx="surname">Fuller</span>
                           <span tagx="given-names">SD</span>
                        </name>
                        <etal></etal>
                     </span>
                     <span tagx="article-title">The Fusion glycoprotein shell of Semliki Forest virus: An icosahedral assembly primed
                        for fusogenic activation at endosomal pH
                     </span>
                     <span tagx="source">Cell</span>
                     <span tagx="year">2001</span>
                     <span tagx="volume">105</span>
                     <span tagx="fpage">137</span>
                     <span tagx="lpage">148</span>
                     <span class="pub-id'">[11301009]</span>
                  </div>
               </li>
            </ul>
         </div>
         <div tagxxx="glossary">
            <title>Abbreviations</title>
            <div tagxxx="def-list">
               <div tagxxx="def-item">
                  <div tagxxx="term">aa</div>
                  <div tagxxx="def">
                     <p>amino acid</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">CHIKV</div>
                  <div tagxxx="def">
                     <p>chikungunya virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">CSF</div>
                  <div tagxxx="def">
                     <p>cerebrospinal fluid</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">FBS</div>
                  <div tagxxx="def">
                     <p>foetal bovine serum</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">HMAF</div>
                  <div tagxxx="def">
                     <p>hyperimmune ascitic fluid</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">ONN</div>
                  <div tagxxx="def">
                     <p>O'nyong-nyong virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">SFV</div>
                  <div tagxxx="def">
                     <p>Semliki Forest virus</p>
                  </div>
               </div>
            </div>
         </div>
         <div tagxxx="fn-group">
            <fn>
               <p>Citation: Schuffenecker I, Iteman I, Michault A, Murri S, Frangeul L, et al. (2006)
                  Genome microevolution of chikungunya viruses causing the Indian Ocean outbreak. PLoS
                  Med 3(7): e263. DOI: 10.1371/journal.pmed.0030263
               </p>
            </fn>
            <fn>
               <p>
                  <b>Funding:</b> Financial support was received from the Institut Pasteur, the Centre National de
                  la Recherche Scientifique (CNRS), the Réseau National des Genopoles (RNG), and the
                  Institut de Veille Sanitaire. The funders had no role in study design, data collection
                  and analysis, decision to publish, or preparation of the manuscript.
                  				
               </p>
            </fn>
         </div>
      </back>
   </article>
</html>